lib/engine.js

```diff
@@ -55,6 +65,21 @@ Engine.prototype.start = function (callback) {
     this._config.storage.mongoOpts,
     { logger: log }
   );
+  this.network_queue = new NetworkMessageQueue({
+    connection: {
+      url: `amqp://${USER}:${PASS}@${HOST}:${PORT}`,
+    },
+    exchange: {
+      name: "exchangeName",
+      type: "direct",
+    },
+    queue: {
+      name: QUEUE_NAME,
+    },
+    routingKey: {
+      name: "routingKeyName",
+    },
```

Comment:
Not sure if this initialisation parameters should be performed here or imported from somewhere else